guides/source/users/products/product-properties.html.md

```diff
@@ -4,8 +4,8 @@ Product properties track individual attributes for a product, especially if the
 attributes only apply to one specific product. Typically, product properties
 would be used for technical specifications or additional production information. 
 
-For example, you can list of product properties for a limited edition t-shirt as
-a table on its product page:
+For example, you t see a list of product properties for a limited edition
```

Comment:
Typo here.

Reply:
This file is no longer being changed on the branch.

I think I messed up a rebase in the past, which caused this change to show up. But I can't remember this far back.